common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -285,7 +287,7 @@ def get_html(self, view=STUDENT_VIEW):
         if getattr(self, 'video_speed_optimizations', True) and cdn_url:
             branding_info = BrandingInfoConfig.get_config().get(self.system.user_location)
 
-            if self.edx_video_id and edxval_api:
+            if self.edx_video_id and edxval_api and video_status != u'external':
```

Comment:
In the case where we have an external video and no transcript (so there is no video id), will video_status always be 'external'?  I'm still wondering about whether we can eliminate the check for edx_video_id and edxval_api.  Logically it seems like video_status is the right way to make this check, if its always accurate.

Reply:
In that scenario, there will be no video status because there will be no VAL entry. The variable will have **None** and will not be updated.

Reply:
and in python, None != 'external', so the condition will be true and re-write will occur.

Reply:
Makes sense, thanks for the explanation!